cmd/end2endtest/lifecycle.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+
+	vapi "go.vocdoni.io/dvote/api"
+	"go.vocdoni.io/dvote/apiclient"
+	"go.vocdoni.io/dvote/log"
+	"go.vocdoni.io/proto/build/go/models"
+)
+
+func init() {
+	ops["lifecyclelection"] = operation{
+		test:        &E2ELifecycleElection{},
+		description: "Publishes a lifecyclelection election to validate that the election cannot be interrupted changing the status to ENDED or PAUSED or CANCELED or RESULTS",
+		example:     os.Args[0] + " --operation=lifecyclelection --votes=1000",
+	}
+}
+
+var _ VochainTest = (*E2ELifecycleElection)(nil)
+
+type e2eElections []e2eElection
+
+type E2ELifecycleElection struct {
+	e2eElections
+}
+
+func (t *E2ELifecycleElection) Setup(api *apiclient.HTTPclient, c *config) error {
```

Comment:
in general i feel this whole Setup is very complex: new structs and types are used, and a whole goroutine architecture to parallelize creating 2 elections (which is something the outer framework already solves)

i'd suggest to keep it simple: make lifecycle.go create a single election with interruptible=false. why do you want to handle two elections at the same time?

Reply:
@p4u suggested it in a early comment handle the two elections in one end2end test named lifecycle